Apps/Log/Tests iOS/Log-iOS.xctestplan

```diff
@@ -0,0 +1,44 @@
+{
+  "configurations" : [
+    {
+      "id" : "0D0B75BA-7CDF-4A89-B68C-2012FAEBF180",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "codeCoverage" : {
+      "targets" : [
+        {
+          "containerPath" : "container:Log.xcodeproj",
+          "identifier" : "E25E251E27C23E1E00A0CCDA",
+          "name" : "Log (iOS)"
+        }
+      ]
+    },
```

Comment:
Will this collect code coverage report for all the targets?

Reply:
You were right, selecting to include all targets removes these lines:
<img width="762" alt="Screen Shot 2022-05-24 at 8 33 48 AM" src="https://user-images.githubusercontent.com/1413987/169964413-26456fa0-350b-43e7-a6e9-5f1e37c70d9e.png">
<img width="757" alt="Screen Shot 2022-05-24 at 8 33 56 AM" src="https://user-images.githubusercontent.com/1413987/169964417-133676c3-e68e-407a-80dd-05b1be3a233e.png">